modules/fluid_properties/include/userobjects/SodiumProperty.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     BISON                     */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     Under Contract No. DE-AC07-05ID14517      */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+#ifndef SODIUMPROPERTY_H
+#define SODIUMPROPERTY_H
+
+#include "FluidProperties.h"
+#include "Moose.h"
+
+class SodiumProperty;
+
+template <>
+InputParameters validParams<SodiumProperty>();
+
+/// Properties of liquid sodium from ANL/RE-95/2 report "Thermodynamic and Transport Properties of
+/// Sodium Liquid and Vapor" from ANL Reactor Engineering Division.
+
+class SodiumProperty : public FluidProperties
+{
+
+public:
+  SodiumProperty(const InputParameters & parameters);
+
+  /**
+   * Thermal conductivity as a function of temperature
+   * @param T temperature
+   */
+  Real thermal_conductivity(Real temperature) const;
+
+  /**
+   * Enthalpy of liquid Na (relative to solid Na at STP) in J/kg as a function of temperature
+   * @param T temperature
+   */
+  Real enthalpy(Real temperature) const;
+
+  /**
+   * Heat capacity of liquid Na in J/kg-K as a function of temperature
+   * @param T temperature
+   */
+  Real heat_capacity(Real temperature) const;
```

Comment:
Is this heat capacity denoted by `C`? Or is this something else?

Reply:
c is speed of sound as far as I remember. That's why heat_capacity is used here.

Reply:
Right, I was referring to [this](https://en.wikipedia.org/wiki/Heat_capacity#Extensive_properties), they use capital `C` for it. I was not suggesting a rename to `C`. If you want to keep it spelled out, then we need to go with `camelCase`, not `c_style`, i.e. `heatCapacity`.  If this is `C_p` or `C_v`, then we would use `cp` or `cv`, respectively.

Reply:
Alright. I'll go with heatCapacity. I just find 'c' and 'C' confusing.